### PR TITLE
Upgrade to Calcite 1.35

### PR DESCRIPTION
**Pull request by mihaibudiu**
I also ordered alphabetically the documentation of the string functions, there are no other changes in that file.

### CODE REVIEW

**Comment by mihaibudiu** [comment]
Lots of adapter tests failed, but I bet none of them is caused by this PR.

File: sql-to-dbsp-compiler/SQL-compiler/src/main/codegen/includes/parserPostgresqlImpls.ftl [hunk 1]
```diff
@@ -1,268 +0,0 @@
-<#--
-// Licensed to the Apache Software Foundation (ASF) under one or more
```
Comment (lalithsuresh) [inline sql-to-dbsp-compiler/SQL-compiler/src/main/codegen/includes/parserPostgresqlImpls.ftl:2]: why is this file no longer needed?

Comment (mihaibudiu) [reply sql-to-dbsp-compiler/SQL-compiler/src/main/codegen/includes/parserPostgresqlImpls.ftl:2]: Last time you asked why it was needed. We actually don't use any of these SQL constructs. You were right, it's not used. It comes from Calcite.

File: docs/src/css/custom.css [hunk 2]
```diff
@@ -10,6 +10,8 @@ Copyright © 2021 Phylo, Dr. Electron, Jeroen van den Hout, Critical
 }
 
 :root {
+  --ifm-code-border-radius: 0;
```
Comment (gz) [inline docs/src/css/custom.css:13]: why?

Comment (mihaibudiu) [reply docs/src/css/custom.css:13]: It looks terrible, and the rounder corners sometimes butt into the text

**Pull request merged**